components/list.stories.tsx

```diff
@@ -1014,7 +1014,11 @@ export const WithBadge: Story = {
                 actions={[
                   {
                     title: "Back",
-                    disabled: !isOpen,
+                    style:
```

Comment:
please use `isOpen && ...` kind of syntax; so please make necessary changes to skip rendering any `null`/`false`/`undefined`